bitnami/moodle/values.yaml

```diff
@@ -253,13 +273,15 @@ affinity: {}
 ##
 nodeSelector: {}
 ## Configure resource requests and limits
-## ref: https://kubernetes.io/docs/user-guide/compute-resources/
-## @param resources.requests [object] CPU/Memory resource requests/limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param resources.requests [object] CPU/Memory resource requests
```

Comment:
```suggestion
## @param resources.requests CPU/Memory resource requests
```

Reply:
Isn't `resources.requests [object]` okay?

Reply:
Since the object is not too big, avoiding `[object]` will add the two params in the README table as follows
```
| {memory: 512Mi, cpu: 300m} |
```

`[object]` or `[array]` are used when there are a lot of parameters or a big string that we want to hide from the README table